style/sass/README.md

```diff
@@ -40,6 +40,7 @@
 
 * Don't use ID's for style.
 * Use meaningful names: `$visual-grid-color` not `$color` or `$vslgrd-clr`.
+* Be consistent about naming conventions for classes. If the project is using BEM continue to use BEM, if the project is not using BEM don't introduce a few BEM classes.
```

Comment:
Not sure the second sentence is needed but I think that's because this the problem we're most likely to deal with? It feels a bit passive-agressive especially the fact that BEM is repeated 4 times.